tests/Application/Console/StyleTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\Application\Console;
+
+use NunoMaduro\PhpInsights\Application\Console\Style;
+use Tests\TestCase;
+
+/**
+ * @covers \NunoMaduro\PhpInsights\Application\Console\Style
+ */
+class StyleTest extends TestCase
+{
+    /** @test */
+    public function string_has_correct_length_when_one_digit_value()
```

Comment:
Could we use the style testStringHasCorrectLengthWhenOneDigitValue? Do you mind?

Reply:
I would add return `void` too.

Reply:
Same goes for the other tests.